src/ui/pages/work/WorkorderScheduling.js

```diff
@@ -4,6 +4,7 @@ import EAMDatePicker from 'eam-components/dist/ui/components/muiinputs/EAMDatePi
 import EAMAutocomplete from "eam-components/dist/ui/components/muiinputs/EAMAutocomplete";
 import WS from "../../../tools/WS";
 import UDFChar from "../../components/userdefinedfields/UDFChar";
+import { Grid } from '@material-ui/core';
```

Comment:
This is not being used, although it would be a preferred way to setup the layout for both fields.